src/components/search/concept-modal/concept-modal.js

```diff
@@ -98,12 +99,78 @@ export const ConceptModalBody = ({ result }) => {
-    'kgs':          { title: 'Knowledge Graphs', icon: <KnowledgeGraphsIcon />,  content: <KnowledgeGraphsTab graphs={ graphs } />, },
-    'explanation':  { title: 'Explanation',      icon: <ExplanationIcon />,      content: <ExplanationTab result={ result } /> },
-    'tranql':       { title: 'TranQL',           icon: <TranQLIcon />,           content: <TranQLTab result={ result } graphs = { graphs } /> }
+    'overview': {
+      title: 'Overview',
+      icon: <OverviewIcon />,
+      content: <OverviewTab result={ result } />,
+      tooltip: <div>
+        The Overview tab provides a brief description of the concept.
+        Hovering over the <InfoCircleOutlined style={{ color: "rgba(0, 0, 0, 0.45)", margin: "0 2px" }} />
+        &nbsp;besides the concept name displays the corresponding ontological
+        identifier, and where applicable, a link to the entry on the ontology&apos;s website.
+      </div>
+    },
+    'studies': {
+      title: studyTitle,
+      icon: <StudiesIcon />,
+      content: <StudiesTab studies={ studies } />,
+      tooltip: <div>
+        The Studies tab displays studies referencing or researching the concept.
+        Studies are grouped into { studySources.length } categories:&nbsp;
+        { studySources.map((source, i) => (
+          <Fragment key={ source }>
+            { i === studySources.length - 1 ? "and " : ""}
+            <span style={{ fontWeight: 500 }}>{ source }</span>
+            { i !== studySources.length - 1 ? ", " : ". " }
+          </Fragment>
+        )) }
+        By default, all studies are shown, but categories can be filtered
+        out by clicking on them. You can also click on studies to view which of their variables are
+        related to the concept.
+      </div>
+    },
+    'cdes': {
+      title: cdeTitle,
+      icon: <CdesIcon />,
+      content: <CdesTab cdes={ cdes } cdeRelatedConcepts={ cdeRelatedConcepts } cdeRelatedStudies={cdeRelatedStudies} loading={ cdesLoading } />,
+      tooltip: <div>
+        The CDEs tab displays{ context.brand === "heal" ? " HEAL-approved" : "" } common data elements (CDEs)
+        associated with the concept. A CDE is a standardized question used across studies and clinical
+        trials with a precisely defined set of permissible responses to ensure consistent data collection.
+        {context.brand === "heal" ? <span>
+          &nbsp;The <a href="https://heal.nih.gov/data/common-data-elements" target="_blank" rel="noopener noreferrer">
+            HEAL CDE program
+          </a> provides further information.
+        </span> : null}
+      </div>
+    },
+    'kgs': {
+      title: 'Knowledge Graphs',
+      icon: <KnowledgeGraphsIcon />, 
+      content: <KnowledgeGraphsTab graphs={ graphs } />,
+      tooltip: <div>
+        The Knowledge Graphs tab displays the <a href="https://www.oxfordsemantic.tech/faqs/what-is-a-knowledge-graph" target="_blank" rel="noopener noreferrer">
```

Comment:
I like the idea of linking to a webpage about knowledge graphs, but I would either link to the Wikipedia page (https://en.wikipedia.org/wiki/Knowledge_graph) or the Neo4J page (https://neo4j.com/blog/what-is-knowledge-graph/), which I think are better. Alternatively, you could change this line to:

```suggestion
        The Knowledge Graphs tab displays relevant edges from the <a href="https://robokop.renci.org/" target="_blank" rel="noopener noreferrer">ROBOKOP Knowledge Graph</a>
```

Reply:
Added fix.